test/ContentClient.spec.ts

```diff
@@ -16,13 +16,52 @@ import {
 } from 'dcl-catalyst-commons'
 import { DeploymentWithMetadataContentAndPointers } from 'ContentAPI'
 import { Headers } from 'node-fetch'
+import { DeploymentBuilder } from 'utils'
 
 chai.use(chaiAsPromised)
 const expect = chai.expect
 
 describe('ContentClient', () => {
   const URL = 'https://url.com'
 
+  describe("When calling buildDeployment", () => {
+    let mocked;
+    let fetcher;
+    const currentTime = 100;
+
+    beforeEach(async () => {
+      ({ mock: mocked, instance: fetcher } = mockFetcherJson('/status', { currentTime }))
+
+      const deploymentBuilderClassMock: typeof DeploymentBuilder = mock<typeof DeploymentBuilder>(DeploymentBuilder);
+
+      const type = EntityType.PROFILE;
+      const pointers = ["p1"];
+      const files = new Map();
+      const metadata = {};
+
+      when(deploymentBuilderClassMock.buildEntity(type, pointers, files, metadata, currentTime)).thenResolve()
+
+      const client = buildClient(URL, fetcher, instance(deploymentBuilderClassMock))
+
+      await client.buildDeployment(type, pointers, files, metadata)
+    })
+
+    it("should fetch the status", () => {
+      verify(mocked.fetchJson(URL + '/status', anything())).once()
```

Comment:
can we check also that the value received by the DeploymentBuilder is the same timestamp?

Reply:
Technically it's verified by the mock since the `when` returns only when the timestamp is the one expected, but I can add a specific check, makes sense